src/widgets/forms.js

```diff
@@ -230,65 +403,26 @@ forms.field[UI.ns.ui('Multiple').uri] = function (
     return box
   }
 
-  // box.appendChild(dom.createElement('h3')).textContent = "Fields:".
   var body = box.appendChild(dom.createElement('tr'))
   var tail = box.appendChild(dom.createElement('tr'))
-
```

Comment:
Please don't do this big moves of functionality in commits that has other big changes - makes it really hard to review what has changed...